src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -140,11 +144,24 @@ const PIAFormPage = () => {
    * @description - Navigate to the intake page upon status change to an
    *                editable status or if the user changes the mode
    */
+
   useEffect(() => {
-    if (checkEditableStatus() && mode === 'edit')
```

Comment:
You're just adding a new check anyways. I would advice the logic stay as is. Just insert a line before these check;
`if (pathname === routes.PIA_NEW) return;`